console/src/main/java/org/eclipse/rdf4j/console/command/Show.java

```diff
@@ -27,41 +29,55 @@
  *
  * @author Dale Visser
  */
-public class Show implements Command {
+public class Show extends ConsoleCommand {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(Show.class);
 
 	private static final String OUTPUT_SEPARATOR = "+----------";
 
-	private final ConsoleIO consoleIO;
-	private final ConsoleState state;
-
+	@Override
+	public  String getName() {
+		return "show";
+	}
+	
+	@Override
+	public String getHelpShort() {
+		return "Displays an overview of various resources";
+	}
+	
+	@Override
+	public String getHelpLong() {
+		return  PrintHelp.USAGE
+			+ "show {r, repositories}   Shows all available repositories\n"
+			+ "show {n, namespaces}     Shows all namespaces\n"
+			+ "show {c, contexts}       Shows all context identifiers\n";
+	}	
+	
 	/**
 	 * Constructor
 	 *
 	 * @param consoleIO
 	 * @param state
 	 */
-	Show(ConsoleIO consoleIO, ConsoleState state) {
-		this.consoleIO = consoleIO;
-		this.state = state;
+	public Show(ConsoleIO consoleIO, ConsoleState state) {
+		super(consoleIO, state);
 	}
 
 	@Override
 	public void execute(String... tokens) {
 		if (tokens.length == 2) {
 			final String target = tokens[1].toLowerCase(Locale.ENGLISH);
 			if ("repositories".equals(target) || "r".equals(target)) {
-				showRepositories();
+					showRepositories();
 			} else if ("namespaces".equals(target) || "n".equals(target)) {
-				showNamespaces();
+					showNamespaces();
 			} else if ("contexts".equals(target) || "c".equals(target)) {
-				showContexts();
+					showContexts();
 			} else {
-				consoleIO.writeError("Unknown target '" + tokens[1] + "'");
+					consoleIO.writeError("Unknown target '" + tokens[1] + "'");
```

Comment:
Looks like a code formatting issue. @barthanssens can you check your indent settings (or if you're using Eclipse, make sure the project is set to use the predefined RDF4J code formatting settings?). Looks to me like it adds a second tab in a conditional block. 

More generally it might be good for a dev who uses eclipse to run reformatting on this PR.